2020-01-Winter/0_arrays_sorting/README.md

```diff
@@ -118,18 +136,53 @@ Go to [Top](#top)
 
 ### 2. SOLUTION 2 TODO :bug:
 
-Source: TODO :bug:
+Source: <a href="https://leetcode.com/problems/search-in-rotated-sorted-array/">LeetCode</a> :bug:
```

Comment:
Consider using native Markdown instead of HTML.

```suggestion
Source: [LeetCode](https://leetcode.com/problems/search-in-rotated-sorted-array/)
```